meson.build

```diff
@@ -117,11 +119,11 @@ nk_options = [
 ]
 nk = subproject('libnkutils', default_options: nk_options)
 nk_subproject_options = nk.get_variable('nk_options')
-foreach o : nk_options + nk_subproject_options
-    if not nk_options.contains(o) or not nk_subproject_options.contains(o)
-        error('You must not change libnkutils options @0@ != @1@'.format('|'.join(nk_options), '|'.join(nk_subproject_options)))
-    endif
-endforeach
+# foreach o : nk_options + nk_subproject_options
+#     if not nk_options.contains(o) or not nk_subproject_options.contains(o)
+#         error('You must not change libnkutils options @0@ != @1@'.format('|'.join(nk_options), '|'.join(nk_subproject_options)))
+#     endif
+# endforeach
```

Comment:
@eli-schwartz Is there a way I can programatically check if meson dist is being run in the meson.build file? I don't like the fact that I have to comment out this code, otherwise I get an error when running `meson dist`.

Error: 
``` 
builddir/meson-private/dist-unpack/rofi-1.7.0-wayland-dev/meson.build:124:8: ERROR: Problem encountered: You must not change libnkutils options bindings=true|xdg-theme=true|git-work-tree=/home/runner/work/rofi/rofi/builddir/meson-private/dist-unpack/rofi-1.7.0-wayland-dev != xdg-theme=true|bindings=true|git-work-tree=/home/runner/work/rofi/rofi
```

Reply:
It's not entirely clear to me what this code is trying to accomplish...

Unfortunately you cannot test in meson.build if a dist is being made, but there are open feature requests for:

https://github.com/mesonbuild/meson/issues/3917
https://github.com/mesonbuild/meson/issues/6866

Reply:
There is a heuristic workaround in the second ticket.

Reply:
Me neither honestly. I might just leave it commented out since it doesn't affect the build.